README.md

```diff
@@ -1,45 +1,229 @@
+                  "question": "What would you say your monthly debt is?",
+                  "source": "http://www.trulia.com"
+              },
+              "id": "5ff7a745be8682d9e5b0e0c4",
+              "type": "Education object"
+          },
+          "mortgage_term": {
+              "attributes": {
+                  "classification": "Mortgage Term",
+                  "description": "Your mortgage term is the number of years you’ll pay on your loan before you fully own your home. ",
+                  "information": "A mortgage is a classification of loan that uses your home as collateral. It’s typically used to buy a home or in refinance situations to secure a more favorable deal and possibly convert equity in your home to cash.",
+                  "note": "Typically, a fixed-rate mortgage runs 30 years and this is the number we will use if you want us to.",
+                  "question": "What mortgage term are you aiming for?",
+                  "source": "https://www.rocketmortgage.com/"
+              },
+              "id": "5ff7a75ebe8682d9e5b0e0c8",
+              "type": "Education object"
+          },
+          "rent": {
+              "attributes": {
+                  "classification": "Rent",
+                  "description": "",
+                  "information": "",
+                  "note": "The imaginative report gives a total home price you can afford based off of your monthly rent.",
+                  "question": "IMAGINATIVE: What is your current monthly rent?",
+                  "source": ""
+              },
+              "id": "5ff7a764be8682d9e5b0e0c9",
+              "type": "Education object"
+          },
+          "zip_code": {
+              "attributes": {
+                  "classification": "ZIP Code",
+                  "description": "Property taxes and interest rates can vary by location. Enter the ZIP code where you're looking for homes and we'll automatically add property taxes and interest rates for your area.",
+                  "information": "A “cost of living analysis” by two large realty companies discovered that living in the suburbs versus living in the city does impact your cost of living – by thousands of dollars. ",
+                  "note": "The term ZIP is an acronym for Zone Improvement Plan",
+                  "question": "What is the zipcode you'd like to live in?",
+                  "source": "http://www.trulia.com/"
+              },
+              "id": "5ff7a73dbe8682d9e5b0e0c3",
+              "type": "Education object"
+          }
+      }
+  }
+  ```
+- `POST http://localhost:5000/api/v1/education/<id>`
+- `GET http://localhost:5000/api/v1/education/<id>`
+- `PUT http://localhost:5000/api/v1/education/<id>`
+- `DELETE http://localhost:5000/api/v1/education/<id>`
 
-## Endpoint Documentation
+### Report Model
+#### REQUEST
+```
+User clicks submit on survey
+# POST http://localhost:5000/api/v1/report
+{
+  salary: 55000,
```

Comment:
And just to confirm, no more mortgage question?

Reply:
When you say "mortgage question", just to confirm you are talking about the mortgage term? If so, yes. I believe that was the discussion during our FS meeting about what questions were required. We decided that since we were assuming 30 years in our BE report that means we don't need to give them a choice to change this. That will come for a registered user (who can ideally select a custom mortgage term && they can select between a fixed-rate and a variable-rate)